src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -48,13 +48,12 @@ var UnoChoice = UnoChoice || ($ => {
     // --- Cascade Parameter
     /**
      * A parameter that references parameters.
-     *
-     * @param paramName parameter name
-     * @param paramElement parameter HTML element
-     * @param randomName randomName given to the parameter
+     * @param paramName {String} parameter name
+     * @param paramElement {HTMLElement} parameter HTML element
+     * @param randomName {String} randomName given to the parameter
      * @param proxy Stapler proxy object that references the CascadeChoiceParameter
      */
-    /* public */ function CascadeParameter(paramName, paramElement, randomName, proxy) {
```

Comment:
The `/* public */` interferes with JSDoc - so I've removed all occurrences of it.